raiden/api/test_encoding.py

```diff
@@ -4,7 +4,7 @@
 from raiden.channel import Channel, ChannelEndState, ChannelExternalState
 from raiden.utils import make_address, make_privkey_address
 
-from raiden.api.encoding import ChannelListSchema, ChannelSchema
```

Comment:
no need for this change. But thanks for actually notifying us of this test file. It should probably be deleted, or at the very least fixed, and put along with all other tests under the `tests/` directory.